core/ulog/doc.go

```diff
@@ -20,32 +20,37 @@
 
 // Package ulog is the Logging package.
```

Comment:
Where does this line come from?

- If we're going to monospace and lowercase ``package ulog``, let's do it in doc.go as well
- No reason to capitalize "logging"
- Instead: "``package ulog`` is UberFx's logging package" for better SEO and reading experience